WordPress/Classes/Services/AccountServiceFacade.m

```diff
@@ -11,7 +11,13 @@ - (WPAccount *)createOrUpdateWordPressComAccountWithUsername:(NSString *)usernam
     NSManagedObjectContext *context = [[ContextManager sharedInstance] mainContext];
     AccountService *accountService = [[AccountService alloc] initWithManagedObjectContext:context];
 
-    return [accountService createOrUpdateAccountWithUsername:username authToken:authToken];
+    WPAccount *account = [accountService createOrUpdateAccountWithUsername:username authToken:authToken];
+    if (![accountService.defaultWordPressComAccount.uuid isEqualToString:account.uuid]) {
```

Comment:
We shouldn't add business logic to facades.  The work being done probably has a better home in `AccountService createOrUpdateAccountWithUsername:authToken:`